Updated index.html

Sumit0581 opened the pull request:
Changed the design of site and made it responsive

Sumit0581 committed:
responsive website

Sumit0581 committed:
responsive css 2

removed some blank spaces

Sumit0581 committed:
responsive css

twitu reviewed (changes requested):
improve readability by using proper indentation
disable/comment background animation for now, keep it a simple solid colour. a few settings changes are required for the images to be loaded from url. I will modify the settings then you can make a separate pull request for background amimation.

twitu commented:
move all .jpg files to separate "media" directory in the project root directory

twitu commented:
also mention the changes you have made in issue #4

Sumit0581 committed:
index.html update

improved readability by using proper indentation
disabled background animation
moved all .jpg files to separate "media" directory in the project root directory
